src/components/createEditCqlLibrary/CreateEditCqlLibrary.tsx

```diff
@@ -244,15 +263,18 @@ const CreateEditCqlLibrary = () => {
 
   const updateElmAnnotations = async (
     cql: string
-  ): Promise<ElmTranslationError[]> => {
+  ): Promise<ValidationResult> => {
     if (cql && cql.trim().length > 0) {
-      const { errors: allErrorsArray } = await validateContent(cql);
-      if (isLoggedInUMLS(allErrorsArray)) {
+      const result = await validateContent(cql);
+      const { errors, externalErrors } = result;
+      // right now we are only displaying the external errors related to included libraries
+      // and only the first error returned by elm translator
+      handleToast("danger", externalErrors[0]?.message, true);
```

Comment:
There is an issue with how this handleToast is being handled, mainly the external errors are not present